packages/odyssey-react-mui/src/Callout.tsx

```diff
@@ -40,20 +52,49 @@ export type CalloutProps = {
    * Determine the color and icon of the Callout
    */
   severity: (typeof calloutSeverityValues)[number];
+  /**
+   * The text content of the Callout
+   */
+  text?: string;
   /**
    * The title of the Callout
    */
   title?: string;
-} & SeleniumProps;
+} & (
+  | {
+      text?: string;
+      children: ReactNode;
+    }
+  | {
+      text: string;
+      children?: ReactNode;
+    }
```

Comment:
Does this actually work? I don't know if TS can actually discern between the two (i.e: is there a state with these options that TS will complain?)